client/verta/docs/tutorials/deployment/endpoint_update_config.rst

```diff
@@ -0,0 +1,70 @@
+Updating Endpoint With a Configuration File
+===========================================
+
+.. TODO: link to Endpoint.update tutorial
+
+Verta's Client and CLI support updating Endpoint with JSON or YAML configuration file. This tutorial will show how it can be done, using the Client and CLI.
+
+Configuration File
+------------------
+
+Sometimes, it is more convenient to specify update configuration in a separate, human-readable file, than to have everything in a Python script or a split CLI command.
+
+For example:
+
+.. code-block:: json
+
+        {
+            "run_id": "<experiment run id>",
+            "strategy": "canary",
+            "canary_strategy": {
+                "progress_step": 0.05,
+                "progress_interval_seconds": 30,
+                "rules": [
+                    {
+                        "rule": "latency_avg_max",
+                        "rule_parameters": [
+                            {
+                                "name": "threshold",
+                                "value": "0.1"
+                            }
+                        ]
+                    },
+                    {
+                        "rule": "error_4xx_rate",
+                        "rule_parameters": [
+                            {
+                                "name": "threshold",
+                                "value": "1"
+                            }
+                        ]
+                    }
+                ]
+            }
+        }
```

Comment:
Would you be able to also squeeze in autoscaling, resources, and environment variables in this example?

Reply:
Yes I can. I have them prepared, but it was getting a bit long, so I cut them out. Will re-add back in.

Reply:
Added.

Reply:
I don't think we can cut them out because I'd like this doc to show the full config file JSON 😆 But you're right that it runs a little long.

@conradoverta What do you think we should do? Cut things from this JSON, or show all of it (with consistent formatting)?

Reply:
It's fine to have everything, but keep in mind that's a maintenance burden. I'd recommend figuring out how we can fetch the format of the full file automatically based on the real API. Otherwise this will become stale as a complete reference in a week or two.